src/frontend/js/components/Form/Inputs.tsx

```diff
@@ -77,15 +79,17 @@ interface TextFieldProps extends React.InputHTMLAttributes<HTMLInputElement>, Fi
 
 export const TextField = forwardRef<HTMLInputElement, TextFieldProps>(
   ({ fieldClasses = [], label, id, type = 'text', error, message, ...props }, ref) => (
-    <Field error={error} message={message} fieldClasses={fieldClasses}>
+    <Field id={id} error={error} message={message} fieldClasses={fieldClasses}>
```

Comment:
What do you think to rename the prop `id` into `inputId` as currently, at first read, I believe you are use the same id twice.

Reply:
Sure, makes sense